pages/data-migration/_meta.json

```diff
@@ -4,6 +4,7 @@
   "json": "JSON",
   "cypherl": "CYPHERL",
   "migrate-from-neo4j": "Migrate from Neo4j",
-  "migrate-from-rdbms": "Migrate from RDBMS",
+  "migrate-from-rdbms": "Migrate from RDBMS using CSV files",
```

Comment:
If we change the name of the page https://memgraph.com/docs/data-migration/migrate-from-rdbms, then this callout should be removed. We will now have a dedicated page about migration using MAGE modules

![image](https://github.com/memgraph/documentation/assets/46486712/f83222c0-9ca1-490b-8f2d-dbf49654f1d4)

Reply:
I didn't find the callout in this PR after the changes. Perhaps I deleted it already